packages/fuselage/src/components/Slider/SliderHead.tsx

```diff
@@ -0,0 +1,48 @@
+import type { LabelHTMLAttributes, OutputHTMLAttributes } from 'react';
+import React from 'react';
+import type { SliderState } from 'react-stately';
+
+import { Label } from '../Label';
+
+type SliderHeadProps = {
+  state: SliderState;
+  labelProps: LabelHTMLAttributes<HTMLLabelElement>;
+  outputProps: OutputHTMLAttributes<HTMLOutputElement>;
+  label?: string;
+  showOutput?: boolean;
+  multiThumb?: boolean;
+};
+export const SliderHead = ({
+  state,
+  labelProps,
+  outputProps,
+  label,
+  showOutput = true,
+  multiThumb,
+}: SliderHeadProps) => {
+  const output = (
+    <Label {...(outputProps as any)}>
+      {multiThumb ? (
+        <Label {...(outputProps as any)} data-testid='slider-output'>
+          {`${state.getThumbValueLabel(0)} - ${state.getThumbValueLabel(1)}`}
+        </Label>
+      ) : (
+        <Label {...(outputProps as any)} data-testid='slider-output'>
+          {state.getThumbValueLabel(0)}
+        </Label>
+      )}
+    </Label>
+  );
+  return (
+    <>
+      {label ? (
+        <div className='rcx-slider-label-container'>
+          <Label {...(labelProps as any)}>{label}</Label>
```

Comment:
**question:** What is up with all these `as any` spreads?

Reply:
`Label` was giving me a ts error because of the `is` prop spread on outputProps
_ Types of property 'is' are incompatible._

Reply:
Done: Omitted 'is' prop